lib/zora/filterCasts.ts

```diff
@@ -0,0 +1,55 @@
+import { Cast } from '@neynar/nodejs-sdk/build/neynar-api/v2';
+import getCastsMetadataLink from './getCastsMetadataLink';
+import getCastContractMapping from './mapCastToContract';
+import getIpfsLink from '../getIpfsLink';
+
+const thirdwebId = process.env.THIRDWEB_CLIENT_ID!;
+
+async function filterZoraFeed(casts: Cast[]) {
+  const batchSize = 99;
+  const castsMetadata: Response[] = [];
+  const response: Cast[] = [];
+
+  try {
+    const data = await getCastsMetadataLink(getCastContractMapping(casts));
+
+    for (let i = 0; i < data.length; i += batchSize) {
+      const batch = data.slice(i, i + batchSize);
+
+      castsMetadata.push(...await Promise.all(
+        batch.map(cast => {
+          return fetch(`https://${thirdwebId}.ipfscdn.io/ipfs/${getIpfsLink(cast.ipfs)}`);
```

Comment:
did you test if this works???
- seems to be stacking 2, unrelated IPFS gateways together...